README.md

```diff
@@ -99,37 +99,65 @@ More clear code style guidelines are probably coming in future.
 If you use Skript as (soft) dependency for your plugin, and use maven or Gradle,
 this is for you.
 
-First, you need the repository. Skript is not available in Maven Central.
-```
-maven {
-    url 'https://raw.githubusercontent.com/bensku/mvn-repo/master'
+First, you need to add the JitPack repository at the **END** of all your repositories. Skript is not available in Maven Central.
+```gradle

```

Comment:
Just put the repository declaration here and mention that it belongs to the repositories block.

If that seems too vague for you, just link them to the [Gradle user guide](https://docs.gradle.org/current/userguide/userguide.html) since they should know where it goes already.

Reply:
JitPack recommends adding its repository at the end which is why I included that instruction.

> Note: when using multiple repositories in build.gradle it is recommended to add JitPack at the end. Gradle will go through all repositories in order until it finds a dependency.

https://jitpack.io/docs/#building-with-jitpack

Reply:
Jitpack can be quite slow at times. If other repo can be used, that is probably a good idea.

Reply:
From my experience the speed of JitPack has been fine. The main issues I see with the current GitHub Repo as a Maven Repo is the extra work required to keep it up to date. Using JitPack removes that extra work, allowing addon developers to always use the latest version of Skript to build.

Reply:
Personally, Jitpack has never been slow to me, and I use it frequently. We're better off just merging this (thing that I already did) and change it if someone happens to have this issue.